src/pages/CriticalIssueEntry.svelte

```diff
@@ -20,6 +20,9 @@
     initAll();
   });
 
+  let defaultStyle =
```

Comment:
I thought about doing the same for the browse page, but because most of our layer styling choices are tuned for the greyscale dataviz one, don't want to change it yet.